src/test/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerXMLInputFileTypeTest.java

```diff
@@ -0,0 +1,39 @@
+package edu.cornell.kfs.module.purap.batch.service.impl;
+
+import edu.cornell.kfs.module.purap.batch.JaggaerXMLInputFileType;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.kuali.kfs.sys.ConfigureContext;
+import org.kuali.kfs.sys.context.KualiIntegTestBase;
+import org.kuali.kfs.sys.context.SpringContext;
+
+@ConfigureContext
+public class JaggaerXMLInputFileTypeTest extends KualiIntegTestBase {
```

Comment:
Did you want to add tests for getAuthorPrincipalName() and getFileName() since those have more interesting code.

Reply:
I added tests for those methods